README.md

```diff
@@ -1,2 +0,0 @@
-Source Code for the Custom Main Menu Minecraft Mod
```

Comment:
Why did you remove this?

Reply:
I just deleted all files in project and created it from scratch. So i lost this file. Restored id.